usaspending_api/database_scripts/etl/award_delta_view.sql

```diff
@@ -1,6 +1,65 @@
 DROP VIEW IF EXISTS award_delta_view;
-
 CREATE VIEW award_delta_view AS
+with closed_periods as (
+    select
+    	distinct concat(w.submission_fiscal_year::text, lpad(w.submission_fiscal_month::text, 2, '0')) AS fyp,
+        w.submission_fiscal_year,
+        w.submission_fiscal_month
+    from dabs_submission_window_schedule w
+    where
+    	-- For COVID, look only after a certain date (likely on/after 2020-04-01, or the start of FY2020 FYP07)
+        w.period_start_date >= '2020-04-01' -- using older date to see data surface
+        and w.submission_reveal_date <= now() -- change "period end" with "window close"
+    order by w.submission_fiscal_year desc, w.submission_fiscal_month desc
+),
+    eligible_submissions AS (
+        SELECT *
+        FROM submission_attributes s
+        WHERE concat(s.reporting_fiscal_year::text, lpad(s.reporting_fiscal_period::text, 2, '0')) IN (
+            SELECT fyp FROM closed_periods
+        )
```

Comment:
Add in `w.is_quarter` and join to it in the `eligible_submissions` CTE as per sql attached to https://federal-spending-transparency.atlassian.net/browse/DEV-5283